src/main/java/com/wolox/training/dtos/BookDTO.java

```diff
@@ -0,0 +1,95 @@
+package com.wolox.training.dtos;
+
+public class BookDTO {
+
+    private int id;
```

Comment:
Why if the id of the book class is a long the dto is integer? we should maintain uniformity of types between the dtos and the classes they represent.